autotest/ogr/ogr_miramon_vector.py

```diff
@@ -1428,3 +1428,47 @@ def test_ogr_miramon_create_field_after_feature(tmp_path):
         match="Cannot create fields to a layer with already existing features in it",
     ):
         create_common_attributes(lyr)
+
+
+###############################################################################
+# test reading mutirecord JSON file. At some point this test failed
+# it's here to check that it's not failing (due to memory problems)
+
+
+def test_ogr_miramon_json_import_not_failing(tmp_vsimem):
+
+    out_filename = str(tmp_vsimem / "out/json_layer_to_mm.pol")
+    gdal.VectorTranslate(
+        out_filename,
+        "data/miramon/reading_test/LT05_L2SP_038037_20120505_20200820_02_T1_ST_stac_minimal.json",
```

Comment:
could you please move the file into a data/miramon_inputs directory ? Because currently having that .json file into a subdirectory of data/miramon/ confuses the fuzzers/build_seed_corpus.sh script

Reply:
Yes, sure. It's done. Thanks for helping.